src/site/stages/build/drupal/health-care-region.js

```diff
@@ -24,6 +24,64 @@ function sortServices(sortItem) {
     .value();
 }
 
+// Creates the past-events listing pages
+function createPastEventListPages(page, drupalPagePath, files) {
+  const sidebar = page.facilitySidebar;
+  // Events listing page
+  const allEvents = page.pastEvents;
+
+  // store past & current events
+  const pastEventTeasers = {
+    entities: [],
+  };
+
+  // separate current events from past events;
+  _.forEach(allEvents.entities, value => {
+    const eventTeaser = value;
+    const startDate = eventTeaser.fieldDate.startDate;
+    const isPast = moment().diff(startDate, 'days');
+    if (isPast >= 1) {
+      pastEventTeasers.entities.push(eventTeaser);
+    }
+  });
+
+  // sort past events into reverse chronological order by start date
+  pastEventTeasers.entities = _.orderBy(
+    pastEventTeasers.entities,
+    ['fieldDate.startDate'],
+    ['desc'],
+  );
+
+  // Past Events listing page
+  const pastEventsEntityUrl = createEntityUrlObj(`${drupalPagePath}`);
```

Comment:
We no longer need the template string.

```suggestion
  const pastEventsEntityUrl = createEntityUrlObj(drupalPagePath);
```

Reply:
Ah, yep :+1: @bkjohnson